virtool/startup.py

```diff
@@ -349,6 +351,26 @@ async def init_version(app: typing.Union[dict, Application]):
     app["version"] = version
 
 
+async def init_b2c(app: Application):
+    b2c_client_id = os.environ.get("B2C_CLIENT_ID")
+    b2c_client_secret = os.environ.get("B2C_CLIENT_SECRET")
+    b2c_tenant = os.environ.get("B2C_TENANT")
+    b2c_user_flow = os.environ.get("B2C_USER_FLOW")
+    b2c_authority = os.environ.get("B2C_AUTHORITY")
+
+    if not b2c_client_id or not b2c_client_secret or not b2c_tenant or not b2c_user_flow or not b2c_authority:
```

Comment:
https://docs.python.org/3/library/functions.html#all
```suggestion
    if not all([b2c_client_id, b2c_client_secret, b2c_tenant, b2c_user_flow, b2c_authority]):
```